force-app/main/default/classes/StringGenerator.cls

```diff
@@ -0,0 +1,30 @@
+public with sharing class StringGenerator {
+
+    private List<String> my_list = new List<String>();
+
+    public StringGenerator( List<String> my_list_by_user )
+    {
+        my_list = my_list_by_user;
+    }
+    
+    public List<String> returnRandomString()
+    {
+        List<Integer> randomIndexList = new List<integer>();
+        
+        List<String> randomStringList = new List<String>();
+        
+        Integer randomIndex;
+        
+        while( randomIndexList.Size() < my_list.Size() )
```

Comment:
sizes cant be equal ?

Reply:
Because we are looking at index values, which will range from 0 to myList.Size()-1

Reply:
let's say we have a list with 5 entries, the indexes will be 0 - 4, but the size of index list will still be 5 - same as values list ?